security_monkey/datastore.py

```diff
@@ -492,6 +495,31 @@ class WatcherConfig(db.Model):
     active = Column(Boolean(), nullable=False)
 
 
+class SplunkAlerter(db.Model):
```

Comment:
Would prefer to not have a specific DB table per plugin.

I do think it's valuable, however, to have a generic plugin configuration table that each plugin can make sense of to store data.

Reply:
Not sure exactly what to do next, what you mean about db table plugins is to have like the config in a json string and parse-it to build the object ?
Like to have some proposition before do anything as the result could be to go to the wrong way.

Reply:
I think this should be a DB table that is somewhat generic, like: `SMPluginConf`.

At a minimum, this would have a unique plugin `name` field, and a `config` field to hold a JSON with whatever configuration your plugin requires.  This way, it can then be liberally used by other Security Monkey plugins -- including this one.